configs/processors/arxiv_unified.yaml

```
@@ -0,0 +1,128 @@
+# ArXiv Unified Pipeline Configuration
+# Properly combines PDF and LaTeX into single documents
+
+# PostgreSQL Configuration (Source of Truth)
+postgresql:
+  host: localhost
+  port: 5432
+  database: Avernus
+  user: postgres
+  # password provided via command line
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)